corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -343,6 +343,11 @@ hqDefine("app_manager/js/details/screen", function () {
             data.persistTileOnForms = self.persistTileOnForms();
             data.persistentCaseTileFromModule = self.persistentCaseTileFromModule();
             data.enableTilePullDown = self.persistTileOnForms() ? self.enableTilePullDown() : false;
+            
+            data.case_tile_group = JSON.stringify({
+                index_identifier: self.caseTileGrouped() ? self.caseTileGroupBy() : null,
```

Comment:
`self.caseTileGrouped() ? self.caseTileGroupBy()` what's the purpose of doing this instead of just `self.caseTileGroupBy() `. It seems to evaluate to `detail.case_tile_group.index_identifier` either way